### PR TITLE
Reintroduce infinispan.version property

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
     <packaging>pom</packaging>
 
     <properties>
+        <infinispan.version>12.1.0-SNAPSHOT</infinispan.version>
```
Comment (wburns) [inline pom.xml:19]: I was thinking for the default we can just have it be `${project.version}` wdyt ?

Comment (wburns) [reply pom.xml:19]: Then the script should never have to touch it. And we only have to change it for one off.s

Comment (ryanemerson) [reply pom.xml:19]: That makes a lot more sense

Comment (ryanemerson) [reply pom.xml]: > Then the script should never have to touch it. And we only have to change it for one offs.

For the release script it was actually easier to always require this value and then just change back to `${project.version}`, so I have done that for now. It just means that there is slight duplication on the input params if the versions are the same.

**Review by wburns — APPROVED** [review]
Look good to me. Just waiting for CI.

File: pom.xml (hunk 1 above)
Comment (wburns) [reply pom.xml]: Alright.

**Comment by wburns** [comment]
Integrated into master, thanks @ryanemerson !

**Pull request merged**